cfgov/unprocessed/apps/analytics-gtm/browserslist

```diff
@@ -8,4 +8,3 @@
 
 > 0.5% in my stats # Sets a threshold to cut off extreme edge case browsers.
 > 5% in US         # Sets a threshold for possible future users.
-Explorer >= 8      # Sets IE8 support to address known New Relic errors.
```

Comment:
Will this drop appropriate support for IE 9 or 10?

Reply:
That's a good question, so possibly it will if those are less than 0.5% of our visitors (or less than 5% in US, as a set). I will re-add this as `Explorer >= 9`.

Reply:
Added as `Explorer >= 9`